static/templates/confirm_dialog/confirm_deactivate_user.hbs

```diff
@@ -18,3 +18,28 @@
         </ul>
     {{/if}}
 </p>
+<label class="checkbox">
+    <input type="checkbox" class="send_email" data-key="{{ key }}" />
+    {{#tr}}
+    Notify this user by email?
+    {{/tr}}
+    <span></span>
+    {{> ../help_link_widget link="/help/deactivate-or-reactivate-a-user#notify-users-of-their-deactivation" }}
+</label>
+<div class="email_field">
+    <p class="border-top">
+        <strong>{{t "Subject" }}:</strong>
+        {{#tr}}
+        Notification of account deactivation on {organization_name}
```

Comment:
We can use `realm_name`; I think you misinterpreted the linter error. Switched this back.